mods/ts/rules/nod-vehicles.yaml

```diff
@@ -256,7 +258,7 @@ STNK:
 		Cost: 1100
 	Tooltip:
 		Name: Stealth Tank
-		Description: Long-range missile tank that can cloak.\nHas weak armor. Can be spotted by infantry.\n  Strong vs Vehicles, Tanks, Aircraft\n  Weak vs Infantry.
+		Description: Weak armoured tank that can cloak.\nCan be spotted by infantry at close range.\n  Strong vs Tanks, Aircraft\n  Weak vs Infantry
```

Comment:
Lightly armored tank with a personal stealth generator.\nArmed with missiles.\nCan be spotted by infantry at close range.